public/hwcbuffer.h

```diff
@@ -38,7 +37,10 @@ struct HwcBuffer {
   int prime_fds_[4];
   uint32_t num_planes_ = 0;
   uint32_t fb_modifiers_[8];
+  uint32_t is_interlaced_ = 0;
   hwcomposer::HWCLayerType usage_ = hwcomposer::kLayerNormal;
+  hwcomposer::HWCDeinterlaceControl interlace_mode_ =
```

Comment:
As talked before, a buffer does not have interlaced_mode, but should have bool is_interlaced

Reply:
I agree with your comment and had initially defined is_interlaced_ as bool since it requires only 8-bits.  However, the function you created, setInterlace() defines "uint32_t interlace" as the second parameter.  That's why it is now uint32_t, in order to be consistent with your setInterlace().  I'm changing to "bool is_interlaced" in HwcBuffer since that makes more sense but leaving setInterlace as is because I'm not aware of the reason behind it.